app/main.py

```diff
@@ -1,44 +1,126 @@
+import uuid
+from typing import List
+
+import pydantic
 from aioredis import Redis, from_url
 from fastapi import Depends, FastAPI, WebSocket, WebSocketDisconnect
+from motor.motor_asyncio import AsyncIOMotorClient, AsyncIOMotorDatabase
+from starlette.middleware.cors import CORSMiddleware
 
-from app.models import Message
-from app.settings import AppSettings
+from .models import Action, ActionType, Message
+from .settings import AppSettings
 
 settings = AppSettings()
 
 
 async def get_redis():
-    redis = await from_url(settings.broker_dsn)
+    redis = await from_url(settings.broker_url)
     yield redis
     await redis.close()
 
 
+async def get_db():
+    client = AsyncIOMotorClient(settings.db_url, uuidRepresentation="standard")
+    yield client[settings.db_name]
+    client.close()
+
+
 app = FastAPI()
 
+app.add_middleware(
+    CORSMiddleware,
```

Comment:
@mrprfrm what is this "CORSMiddleware"?

Reply:
CORS is Cross Origin Resource Sharing, we need it to make possible using current API with browser clients.